lib/rmt/cli/base.rb

```diff
@@ -1,3 +1,5 @@
+require 'rmt/lockfile'
```

Comment:
not sure if I should put it there or in the place where we actually use it. this will be in 2 places then :thinking: